Correct `periodic: bool` value in dataset metadata files`

leothomas opened the pull request:
This PR adds a `periodic:bool = True` value to the Dataset model and `"periodic": false` key to the following manifest files: 
- `detections-ship.json`
- `water-spm.json`
- `water-cholorphyll.json`

### Validation Testing: 
The only difference I've found between the `/datasets` endpoint values (see below) and the `layers/index.js` file in the front-end is: 
- `water-cholorphyll` is listed as a dataset for the `tk` spotlight in the `/datasets` endpoing, but not in the `layers/index.js` file
- `index.js` considers that all global datasets are also spotlight datasets whereas `/datasets` endpoint does not (ie: `datasets/tk` does NOT return `co2` and `co2-diff`). Is this the desired behaviour? 

### `/datasets` endpoint values:
```
GET /datasets/global -->
- population: domain [ ]
- agriculture: domain [2018-03-06T00:00:00 , 2020-06-28T00:00:00 (periodic)
- co2: domain [2020-01-01T00:00:00 , 2020-05-17T00:00:00 (periodic)
- no2-diff: domain [2015-01-01T00:00:00 , 2020-07-01T00:00:00 (periodic)
- co2-diff: domain [2020-01-01T00:00:00 , 2020-05-17T00:00:00 (periodic)
- no2: domain [2004-10-01T00:00:00 , 2020-07-01T00:00:00 (periodic)

GET /datasets/be -->
- nightlights-hd: domain [2020-01-01T00:00:00 , 2020-07-01T00:00:00 (periodic)
- nightlights-viirs: domain [2020-01-01T00:00:00 , 2020-07-08T00:00:00 (periodic)
- detections-plane: domain [2020-01-09T00:00:00 , 2020-05-19T00:00:00 (periodic)

GET /datasets/du -->
- nightlights-hd: domain [2020-01-01T00:00:00 , 2020-07-01T00:00:00 (periodic)
- nightlights-viirs: domain [2020-01-01T00:00:00 , 2020-07-08T00:00:00 (periodic)

GET /datasets/gh -->
- nightlights-hd: domain [2020-01-01T00:00:00 , 2020-07-01T00:00:00 (periodic)
- nightlights-viirs: domain [2020-01-01T00:00:00 , 2020-07-08T00:00:00 (periodic)

GET /datasets/la -->
- nightlights-hd: domain [2020-01-01T00:00:00 , 2020-07-01T00:00:00 (periodic)
- nightlights-viirs: domain [2020-01-01T00:00:00 , 2020-07-08T00:00:00 (periodic)
- detections-plane: domain [2020-01-09T00:00:00 , 2020-05-19T00:00:00 (periodic)
- detections-ship: domain [2020-01-22T00:00:00 , ..., 2020-04-24T00:00:00 (NON periodic)

GET /datasets/sf -->
- water-spm: domain [2020-03-02T00:00:00 , ..., 2020-07-03T00:00:00 (NON periodic)
- nightlights-hd: domain [2020-01-01T00:00:00 , 2020-07-01T00:00:00 (periodic)
- nightlights-viirs: domain [2020-01-01T00:00:00 , 2020-07-08T00:00:00 (periodic)
- water-chlorophyll: domain [2020-03-02T00:00:00 , ..., 2020-07-13T00:00:00 (NON periodic)
- detections-plane: domain [2020-01-10T00:00:00 , 2020-05-19T00:00:00 (periodic)
- detections-ship: domain [2020-01-22T00:00:00 , ..., 2020-04-21T00:00:00 (NON periodic)

GET /datasets/tk -->
- nightlights-hd: domain [2020-01-01T00:00:00 , 2020-07-01T00:00:00 (periodic)
- nightlights-viirs: domain [2020-01-01T00:00:00 , 2020-07-08T00:00:00 (periodic)
- water-chlorophyll: domain [2018-01-03T00:00:00 , ..., 2020-07-25T00:00:00 (NON periodic)
- detections-plane: domain [2020-01-09T00:00:00 , 2020-05-17T00:00:00 (periodic)

GET /datasets/ny -->
- water-spm: domain [2020-01-01T00:00:00 , ..., 2020-08-05T00:00:00 (NON periodic)
- nightlights-hd: domain [2020-01-01T00:00:00 , 2020-07-01T00:00:00 (periodic)
- nightlights-viirs: domain [2020-01-01T00:00:00 , 2020-07-08T00:00:00 (periodic)
- water-chlorophyll: domain [2020-01-01T00:00:00 , ..., 2020-08-05T00:00:00 (NON periodic)
- detections-plane: domain [2020-01-09T00:00:00 , 2020-05-19T00:00:00 (periodic)
- detections-ship: domain [2020-01-20T00:00:00 , ..., 2020-03-27T00:00:00 (NON periodic)
```

drewbo commented on covid_api/db/static/datasets/agriculture.json:
```diff
@@ -3,6 +3,7 @@
     "name": "Agriculture",
     "type": "raster-timeseries",
     "s3_location": "agriculture",
+    "time_unit": "day",
```
I think agriculture is monthly

leothomas replied:
![image](https://user-images.githubusercontent.com/11858457/90795055-97c1d800-e2db-11ea-9134-20dee0647567.png)
Date format looks daily (YYYY_MM_DD), but the `CropMonitor` products look like they fall on the 28th of each month

drewbo replied:
ah, let's collapse those to month names only then

drewbo commented:
@leothomas excellent validation and notes on this PR:
- Tokyo should have chlorophyll so this is correct
- Global datasets should be returned in spotlight areas (need to update)
- Agriculture has an odd issue, I can cover more via slack if you'd like but basically:
  - There are three separate products in the folder `CropMonitor` (monthly, periodic, from UMD), `ALOS2` NDVI from ALOS-2/JAXA, currently only used by ESA dashboard, `gcom-c` NDVI from GCOM/JAXA, also currently only used by ESA dashboard
  - Because I don't want to break another dashboard, I think we should leave the JAXA products as is and move the CropMonitor product to a new folder (and update that datasets `s3_location`)

leothomas commented:
@drewbo 
- updated `time_unit` for Agriculture dataset to `month`
- copied all `CropMonitor*` data to `s3://covid-eo-data/agriculture-cropmonitor`(will have to be removed)
- updated `s3_location` and `tile` in `agriculture.json` to point to `s3://covid-eo-data/agriculture-cropmonitor`
- added global datasets to return for all spotlights (see below for endpoint output)

### `/datasets` output: 
```
GET /datasets/global -->
- population: domain [ ]
- agriculture: domain [2020-01-28T00:00:00 , 2020-06-28T00:00:00 (periodic)
- co2: domain [2020-01-01T00:00:00 , 2020-05-17T00:00:00 (periodic)
- no2-diff: domain [2015-01-01T00:00:00 , 2020-07-01T00:00:00 (periodic)
- co2-diff: domain [2020-01-01T00:00:00 , 2020-05-17T00:00:00 (periodic)
- no2: domain [2004-10-01T00:00:00 , 2020-07-01T00:00:00 (periodic)

GET /datasets/tk -->
- nightlights-hd: domain [2020-01-01T00:00:00 , 2020-07-01T00:00:00 (periodic)
- nightlights-viirs: domain [2020-01-01T00:00:00 , 2020-07-08T00:00:00 (periodic)
- water-chlorophyll: domain [2018-01-03T00:00:00 , ..., 2020-07-25T00:00:00 (NON periodic)
- detections-plane: domain [2020-01-09T00:00:00 , 2020-05-17T00:00:00 (periodic)
- population: domain [ ]
- agriculture: domain [2020-01-28T00:00:00 , 2020-06-28T00:00:00 (periodic)
- co2: domain [2020-01-01T00:00:00 , 2020-05-17T00:00:00 (periodic)
- no2-diff: domain [2015-01-01T00:00:00 , 2020-07-01T00:00:00 (periodic)
- co2-diff: domain [2020-01-01T00:00:00 , 2020-05-17T00:00:00 (periodic)
- no2: domain [2004-10-01T00:00:00 , 2020-07-01T00:00:00 (periodic)

GET /datasets/du -->
- nightlights-hd: domain [2020-01-01T00:00:00 , 2020-07-01T00:00:00 (periodic)
- nightlights-viirs: domain [2020-01-01T00:00:00 , 2020-07-08T00:00:00 (periodic)
- population: domain [ ]
- agriculture: domain [2020-01-28T00:00:00 , 2020-06-28T00:00:00 (periodic)
- co2: domain [2020-01-01T00:00:00 , 2020-05-17T00:00:00 (periodic)
- no2-diff: domain [2015-01-01T00:00:00 , 2020-07-01T00:00:00 (periodic)
- co2-diff: domain [2020-01-01T00:00:00 , 2020-05-17T00:00:00 (periodic)
- no2: domain [2004-10-01T00:00:00 , 2020-07-01T00:00:00 (periodic)

GET /datasets/la -->
- nightlights-hd: domain [2020-01-01T00:00:00 , 2020-07-01T00:00:00 (periodic)
- nightlights-viirs: domain [2020-01-01T00:00:00 , 2020-07-08T00:00:00 (periodic)
- detections-plane: domain [2020-01-09T00:00:00 , 2020-05-19T00:00:00 (periodic)
- detections-ship: domain [2020-01-22T00:00:00 , ..., 2020-04-24T00:00:00 (NON periodic)
- population: domain [ ]
- agriculture: domain [2020-01-28T00:00:00 , 2020-06-28T00:00:00 (periodic)
- co2: domain [2020-01-01T00:00:00 , 2020-05-17T00:00:00 (periodic)
- no2-diff: domain [2015-01-01T00:00:00 , 2020-07-01T00:00:00 (periodic)
- co2-diff: domain [2020-01-01T00:00:00 , 2020-05-17T00:00:00 (periodic)
- no2: domain [2004-10-01T00:00:00 , 2020-07-01T00:00:00 (periodic)

GET /datasets/gh -->
- nightlights-hd: domain [2020-01-01T00:00:00 , 2020-07-01T00:00:00 (periodic)
- nightlights-viirs: domain [2020-01-01T00:00:00 , 2020-07-08T00:00:00 (periodic)
- population: domain [ ]
- agriculture: domain [2020-01-28T00:00:00 , 2020-06-28T00:00:00 (periodic)
- co2: domain [2020-01-01T00:00:00 , 2020-05-17T00:00:00 (periodic)
- no2-diff: domain [2015-01-01T00:00:00 , 2020-07-01T00:00:00 (periodic)
- co2-diff: domain [2020-01-01T00:00:00 , 2020-05-17T00:00:00 (periodic)
- no2: domain [2004-10-01T00:00:00 , 2020-07-01T00:00:00 (periodic)

GET /datasets/sf -->
- water-spm: domain [2020-03-02T00:00:00 , ..., 2020-07-03T00:00:00 (NON periodic)
- nightlights-hd: domain [2020-01-01T00:00:00 , 2020-07-01T00:00:00 (periodic)
- nightlights-viirs: domain [2020-01-01T00:00:00 , 2020-07-08T00:00:00 (periodic)
- water-chlorophyll: domain [2020-03-02T00:00:00 , ..., 2020-07-13T00:00:00 (NON periodic)
- detections-plane: domain [2020-01-10T00:00:00 , 2020-05-19T00:00:00 (periodic)
- detections-ship: domain [2020-01-22T00:00:00 , ..., 2020-04-21T00:00:00 (NON periodic)
- population: domain [ ]
- agriculture: domain [2020-01-28T00:00:00 , 2020-06-28T00:00:00 (periodic)
- co2: domain [2020-01-01T00:00:00 , 2020-05-17T00:00:00 (periodic)
- no2-diff: domain [2015-01-01T00:00:00 , 2020-07-01T00:00:00 (periodic)
- co2-diff: domain [2020-01-01T00:00:00 , 2020-05-17T00:00:00 (periodic)
- no2: domain [2004-10-01T00:00:00 , 2020-07-01T00:00:00 (periodic)

GET /datasets/ny -->
- water-spm: domain [2020-01-01T00:00:00 , ..., 2020-08-05T00:00:00 (NON periodic)
- nightlights-hd: domain [2020-01-01T00:00:00 , 2020-07-01T00:00:00 (periodic)
- nightlights-viirs: domain [2020-01-01T00:00:00 , 2020-07-08T00:00:00 (periodic)
- water-chlorophyll: domain [2020-01-01T00:00:00 , ..., 2020-08-05T00:00:00 (NON periodic)
- detections-plane: domain [2020-01-09T00:00:00 , 2020-05-19T00:00:00 (periodic)
- detections-ship: domain [2020-01-20T00:00:00 , ..., 2020-03-27T00:00:00 (NON periodic)
- population: domain [ ]
- agriculture: domain [2020-01-28T00:00:00 , 2020-06-28T00:00:00 (periodic)
- co2: domain [2020-01-01T00:00:00 , 2020-05-17T00:00:00 (periodic)
- no2-diff: domain [2015-01-01T00:00:00 , 2020-07-01T00:00:00 (periodic)
- co2-diff: domain [2020-01-01T00:00:00 , 2020-05-17T00:00:00 (periodic)
- no2: domain [2004-10-01T00:00:00 , 2020-07-01T00:00:00 (periodic)

GET /datasets/be -->
- nightlights-hd: domain [2020-01-01T00:00:00 , 2020-07-01T00:00:00 (periodic)
- nightlights-viirs: domain [2020-01-01T00:00:00 , 2020-07-08T00:00:00 (periodic)
- detections-plane: domain [2020-01-09T00:00:00 , 2020-05-19T00:00:00 (periodic)
- population: domain [ ]
- agriculture: domain [2020-01-28T00:00:00 , 2020-06-28T00:00:00 (periodic)
- co2: domain [2020-01-01T00:00:00 , 2020-05-17T00:00:00 (periodic)
- no2-diff: domain [2015-01-01T00:00:00 , 2020-07-01T00:00:00 (periodic)
- co2-diff: domain [2020-01-01T00:00:00 , 2020-05-17T00:00:00 (periodic)
- no2: domain [2004-10-01T00:00:00 , 2020-07-01T00:00:00 (periodic)
```
